sysusers.d/dbus.conf

```diff
@@ -0,0 +1,8 @@
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+u dbus - "System Message Bus"
```

Comment:
Normally, systemd does the "fix things where they are broken" route. I think this should be followed up here. At the very least this file should mention the (upstream) dbus bug report requesting they add the sysusers file, so that there is a reference to know when this file is safe to be dropped.